<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [gpodder-devel] ImportError: cannot import name util
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/gpodder-devel/2007-August/index.html" >
   <LINK REL="made" HREF="mailto:gpodder-devel%40lists.berlios.de?Subject=Re%3A%20%5Bgpodder-devel%5D%20ImportError%3A%20cannot%20import%20name%20util&In-Reply-To=%3C46C4477C.2080506%40gmail.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000490.html">
   <LINK REL="Next"  HREF="000497.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[gpodder-devel] ImportError: cannot import name util</H1>
    <B>Pieter De Decker</B> 
    <A HREF="mailto:gpodder-devel%40lists.berlios.de?Subject=Re%3A%20%5Bgpodder-devel%5D%20ImportError%3A%20cannot%20import%20name%20util&In-Reply-To=%3C46C4477C.2080506%40gmail.com%3E"
       TITLE="[gpodder-devel] ImportError: cannot import name util">pdedecker at gmail.com
       </A><BR>
    <I>Thu Aug 16 14:47:56 CEST 2007</I>
    <P><UL>
        <LI>Previous message: <A HREF="000490.html">[gpodder-devel] ImportError: cannot import name util
</A></li>
        <LI>Next message: <A HREF="000497.html">[gpodder-devel] ImportError: cannot import name util
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#491">[ date ]</a>
              <a href="thread.html#491">[ thread ]</a>
              <a href="subject.html#491">[ subject ]</a>
              <a href="author.html#491">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>That worked. Thanks! This is strange though:

-----
(ipodsync) PyID3 not found - falling back to channel cover for iPod 
cover art
-----

But I did install python-id3 v1.2-6.2 on my computer! What could be 
wrong? I already tried reinstalling python-id3.

Pieter De Decker

---
Alistair Sutton schreef:
&gt;<i> On 16/08/07, Pieter De Decker &lt;<A HREF="https://lists.berlios.de/mailman/listinfo/gpodder-devel">pdedecker at gmail.com</A>&gt; wrote:
</I>&gt;<i>   
</I>&gt;&gt;<i> I'm using revision 368 and the version of python I'm using is 2.5.1.
</I>&gt;&gt;<i>
</I>&gt;&gt;<i> ----------
</I>&gt;&gt;<i> <A HREF="https://lists.berlios.de/mailman/listinfo/gpodder-devel">pdedecker at BETAubuntu</A>:~/gpodderSVN$ svn update
</I>&gt;&gt;<i> At revision 368.
</I>&gt;&gt;<i>     
</I>&gt;<i>
</I>&gt;<i> I had trouble up until last night running trunk.
</I>&gt;<i>
</I>&gt;<i> In the end I moved the trunk directory out the way and allowed &quot;svn
</I>&gt;<i> update&quot; to pull in a fresh copy. Everything worked fine after that so
</I>&gt;<i> I can only assume some files didn't get updated properly when Thomas
</I>&gt;<i> changed gpodder.py to gui.py.
</I>&gt;<i>
</I>&gt;<i> Perhaps checking out a fresh copy is worth a try?
</I>&gt;<i>
</I>&gt;<i> Al
</I>&gt;<i>   
</I>

</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000490.html">[gpodder-devel] ImportError: cannot import name util
</A></li>
	<LI>Next message: <A HREF="000497.html">[gpodder-devel] ImportError: cannot import name util
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#491">[ date ]</a>
              <a href="thread.html#491">[ thread ]</a>
              <a href="subject.html#491">[ subject ]</a>
              <a href="author.html#491">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/gpodder-devel">More information about the gpodder-devel
mailing list</a><br>
</body></html>
